test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -65,7 +65,7 @@ public void execute_noPersonDisplayed_returnsInvalidIndexMessage() {
     public void execute_targetPersonNotInAddressBook_returnsPersonNotFoundMessage()
             throws IllegalValueException {
         Person notInAddressBookPerson = new Person(new Name("Not In Book"), new Phone("63331444", false),
-                new Email("notin@book.com", false), new Address("156D Grant Road", false), new UniqueTagList());
+                new Email("notin@book.com", false), new Address("156D Grant Road, #12-34, 231534", false), new UniqueTagList());
```

Comment:
Do note that if you are separating the parameters into multiple lines, the parameters should be aligned with each other. Sometimes the formatting style in IntelliJ is slightly different from what we expect.